setup.py

```diff
@@ -1,27 +0,0 @@
-import shutil
```

Comment:
we still need to keep this file so that github can detect the pypi repo

Reply:
idk if the github dependency graph support pyproject.toml

Reply:
Yes it does